src/components/Modules/components/ModulesListRows.tsx

```diff
@@ -186,26 +187,41 @@ export const ModulesListRows = ({
     // Version
     moduleStatus?.version || EMPTY_TEXT_PLACEHOLDER,
     // Module State
-    <ModuleStatus key="module-state" resource={moduleStatus} />,
+    <ModuleStatus
+      key={`module-state-${resource.name}`}
+      resource={moduleStatus}
+    />,
     // Installation State
-    <StatusBadge
-      key="installation-state"
-      resourceKind="kymas"
-      type={resolveType(
-        kymaResource
+    <>
+      <StatusBadge
+        key={`installation-state-${resource.name}`}
+        resourceKind="kymas"
+        type={resolveType(
+          kymaResource
+            ? resolvedInstallationStateName
+            : managerResourceState?.state ?? '',
+        )}
+        tooltipContent={
+          kymaResource
+            ? moduleStatus?.message ?? managerResourceState?.message
+            : managerResourceState?.message
+        }
+      >
+        {kymaResource
           ? resolvedInstallationStateName
-          : managerResourceState?.state ?? '',
+          : managerResourceState?.state}
+      </StatusBadge>
+      {moduleStatus?.maintenance === true && (
```

Comment:
Badges are not shown fully
<img width="199" height="224" alt="Screenshot 2025-08-28 at 08 44 35" src="https://github.com/user-attachments/assets/09c479fd-7a18-4b5b-acaa-4046f1fed044" />
<img width="255" height="625" alt="Screenshot 2025-08-28 at 10 20 36" src="https://github.com/user-attachments/assets/c1ed648f-7c60-4246-9491-ec2b4348cbbf" />

Reply:
Maybe we can have badge with just the icon on smaller screens? 
<img width="47" height="33" alt="Screenshot 2025-08-28 at 10 28 49" src="https://github.com/user-attachments/assets/23a6c3d2-aa70-40a8-b2e8-7ea7964196dd" />